analyses/molecular-subtyping-pathology/clinical-subtyping-glialneuronal-tumors.Rmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Using WHO 2016 CNS subtypes to improve neuronal and mixed neuronal-glial tumors harmonized diagnosis"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: JN Taroni for ALSF CCDL (code)
+date: 2021
+---
+
+Neuronal and mixed neuronal-glial tumors have subtypes per the [WHO 2016 CNS subtypes](https://link.springer.com/content/pdf/10.1007/s00401-016-1545-1.pdf). 
+However, these are not captured in our molecular data.
+Instead, we can use the pathology free text information in the histologies file to further classify neuronal and mixed neuronal-glial tumors.
+We will use this notebook to do so; see [#996](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/996) for more information.
+
+## Set Up
+
+### Libraries
+
+```{r}
+library(tidyverse)
+```
+
+### Input
+
+```{r}
+data_dir <- file.path("..", "..", "data")
+histologies_file <- file.path(data_dir, "pbta-histologies.tsv")
```

Comment:
```suggestion
histologies_file <- file.path(data_dir, "pbta-histologies-base.tsv")
```

I believe the scripts here should use `pbta-histologies-base.tsv` since the `pbta-histologies.tsv` is only created later in `molecular-subtyping-integrate` module.

Reply:
I think that's probably true _here_ but not in #1016 (https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1016/files#r625098165) since that's downstream of CRANIO subtyping... does that seem correct to you?

Reply:
Addressed in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1017/commits/da8ae7b2a2dede20b7f6c30b648c2e440caccf81